pkg/snapstore/s3_snapstore.go

```diff
@@ -577,6 +577,9 @@ func (s *S3SnapStore) List(_ bool) (brtypes.SnapList, error) {
 		// allSnapKeysMapToCreationTime contains oldest snapshots keys mapped to their versionID and creation timestamp.
 		allSnapKeyMapToSnapshotInfo := make(map[string]*snapshotMetaInfo)
 
```

Comment:
```suggestion
		// allSnapKeyMapToSnapshotInfo contains oldest snapshots keys mapped to their versionID and creation timestamp.
		allSnapKeyMapToSnapshotInfo := make(map[string]*snapshotMetaInfo)

```
The constant `allSnapKeysMapToCreationTime` doesn't exist anywhere. I guess you changed the name after that but missed the comment.